app/plugins/mixinCommonMethods.js

```diff
@@ -104,6 +128,49 @@ export function getLanguageGraph (type) {
   }
 }
 
+// Input: all active arcsi shows
+// Output: shows grouped by days (every day is an array) and sorted tarting today's day (CET)
+export function groupShowsByDay (dbshows, rare_thu, rare_fri, customSchedule) {
```

Comment:
There is an [API](https://arcsi.lahmacun.hu/doc/#/Show%20Requests%20for%20frontend/get_show_schedule_by_day__day_) for querying the daily schedule, it's not handling the rare shows right now, but maybe we can use that for this use case.

Reply:
Absolutely, we could save a lot of computation. Future work?

Reply:
deal